src/dataExplorer/components/Sidebar.tsx

```diff
@@ -125,7 +122,7 @@ const Sidebar: FC = () => {
       justifyContent={JustifyContent.FlexStart}
       className="container-right-side-bar"
     >
-      {isIoxOrg && shouldNotShowSidebar ? null : (
+      {shouldNotShowSidebar ? null : (
```

Comment:
The visibility of the Sidebar component is not depend on org type but depend on language type which is defined in `shouldNotShowSidebar`